apps/web/src/common/constants.ts

```diff
@@ -36,3 +36,6 @@ export const sidebarNavItems: SidebarButtonProps[] = [
     faIcon: faUsers,
   },
 ];
+
+export const steps = ["Profile", "Care Activities Bundles", "Occupation", "Activities Gap", "Suggestions"];
```

Comment:
Could you rename this steps as PlanningSteps or something

Reply:
DOne